### PR TITLE
1.6.0.0

### PR DESCRIPTION
**Pull request by bezirg**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Changelog fragments have been written (if appropriate)
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting master unless this is a cherry-pick backport
    - [x] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

**Review by zliu41 — APPROVED** [review]

**Review by michaelpj — APPROVED** [review]

File: prettyprinter-configurable/prettyprinter-configurable.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version:      2.4
 name:               prettyprinter-configurable
-version:            1.1.0.0
+version:            1.6.0.0
```
Comment (michaelpj) [inline prettyprinter-configurable/prettyprinter-configurable.cabal:3]: Why? This isn't in the list of packages we release, normally?

**Review by michaelpj — CHANGES_REQUESTED** [review]
Need to clarify what's going on with prettyprinter-configurable

**Comment by bezirg** [comment]
> Need to clarify what's going on with prettyprinter-configurable

Its code was touched 2 days ago. It was a code refactor (language pragma cleanup), so it is not 100% sure that is benign. So I decided to bump its version

**Comment by bezirg** [comment]
> Need to clarify what's going on with prettyprinter-configurable

What would you like to be clarified?

**Comment by bezirg** [comment]
@michaelpj @zliu41 I will create a new PR to document the changes to RELEASE process

**Pull request merged**